base/audit_events.go

```diff
@@ -25,6 +25,21 @@ const (
 	AuditIDPublicUserAuthenticationFailed AuditID = 53261
 
 	AuditIDReadDatabase AuditID = 53301
+
+	// BLIP Replication events
+	AuditIDReplicationConnect    AuditID = 54300
```

Comment:
Sorry, I somehow didn't realise I'd not committed the list of events that I had previously shared! I've just opened a PR #6950 to do that.

If we can get that merged, I'll be able to review your changes vs. the original event definitions.